contracts/reserve/ExchangeHelper.sol

```diff
@@ -228,31 +232,27 @@ contract ExchangeHelper is DAOUpgradeableContract {
 			address(_sellTo) != daiAddress && address(_sellTo) != cDaiAddress
 		) {
 			result = _redeemDAI(result);
-			address[] memory path = new address[](2);
 
-			Uniswap uniswapContract = Uniswap(
-				nameService.getAddress("UNISWAP_ROUTER")
-			);
 			uint256[] memory swap;
 			if (address(_sellTo) == address(0x0)) {
-				path[0] = daiAddress;
-				path[1] = uniswapContract.WETH();
-				swap = uniswapContract.swapExactTokensForETH(
+				swap = _uniswapSwap(
+					uniswapContract,
+					_sellTo,
 					result,
+					false,
+					0,
 					_minTokenReturn,
-					path,
-					receiver,
-					block.timestamp
+					receiver
 				);
 			} else {
-				path[0] = daiAddress;
-				path[1] = address(_sellTo);
-				swap = uniswapContract.swapExactTokensForTokens(
+				swap = _uniswapSwap(
+					uniswapContract,
+					_sellTo,
 					result,
+					false,
+					0,
 					_minTokenReturn,
-					path,
-					receiver,
-					block.timestamp
+					receiver
```

Comment:
i dont see the difference between this two calls

Reply:
Yeah right, removed one of those